e2e/cypress.json

```diff
@@ -2,6 +2,7 @@
     "baseUrl": "http://localhost:8065",
     "defaultCommandTimeout": 20000,
     "ignoreTestFiles": "**/node_modules/**/*",
+    "chromeWebSecurity": false,
```

Comment:
Please move up the `chromeWebSecurity` just below the `baseUrl` to main alphabetical order (except for `env` which we wanted to be last).

Please add CWS values in env options so we can use the default in local and have the ability to update in production tests.
```
"env": {
    ... (maintain alpha order as well),
    "cwsUrl": "http://localhost:3000",
    "cwsApiUrl": "http://localhost:8076",
```

In your local setup, you may manually change it to:
```
"cwsUrl": "http://cws-test-server.dev.spinmint.com:8076",
"cwsApiUrl": "http://cws-test-server.dev.spinmint.com:8076",
```

Reply:
updated the place of "chromeWebSecurity" key  in cypress.json file